### PR TITLE
[Fleet] Update CODEOWNERS for Fleet/Agent resources

### PR DESCRIPTION
**Pull request by kpollich**
Point Fleet/Agent ES files to Fleet team instead of Agent Control Plane team.

### CODE REVIEW

**Comment by kpollich** [comment]
image

Not sure why GitHub seems to think the @elastic/fleet team doesn't exist here.

**Review by jlind23 — APPROVED** [review]
Please make sure that the fleet team has write access to this repository.

**Comment by kpollich** [comment]
I don't have admin permissions to `elastic/elasticsearch` in order to add write access for the Fleet team, so we'll need help from the ES team here.

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -17,5 +17,5 @@ x-pack/plugin/core/src/main/resources/monitoring-mb-ilm-policy.json @elastic/inf
 x-pack/plugin/monitoring/src/main/java/org/elasticsearch/xpack/monitoring/MonitoringTemplateRegistry.java @elastic/infra-monitoring-ui
 
 # Elastic Agent
```
Comment (joshdover) [inline .github/CODEOWNERS:19]:
```suggestion
# Fleet
```

**Review by joshdover — APPROVED** [review]

**Comment by mark-vieira** [comment]
@kpollich Sorry I'm just now seeing this. Is this still relevant?

**Comment by kpollich** [comment]
Hi @mark-vieira - no worries at all. Yes this is still relevant for us.

**Comment by kpollich** [comment]
@elasticmachine merge upstream

**Pull request merged**